### DISCUSSION TITLE
Clarify how to update `npm init` packages

### DISCUSSION THREAD

**Discussion by bennypowers** [npm/feedback#163] [CLI] [answered]
I have an old version of my init package `@apollo-elements/create` lodged in my system somewhere. When I run `npm init @apollo-elements`, I get the old version's output. It wasn't clear to me how to update the version. Using `npx @apollo-elements/create@latest` helped, but then subsequently running `npm init @apollo-elements` reverted to the old version. I didn't find any copies of the package in `~/.npm` or in the `node_modules` of my nvm-installed node versions.

What's the official way to clear that cache or update a version of a previously-installed init package? 

```bash
rm -rf ~/.npm/_npx
```
?

**Comment by ljharb** [comment]
Any chance it’s installed or linked globally?

**Comment by bennypowers** [comment]
```npm r -g @apollo-elements```had no effect

**Reply by ljharb** [reply]
Can you check `npm ls -g --depth=0`?

**Reply by bennypowers** [reply]
```
/Users/bennyp/.nvm/versions/node/v15.5.1/lib
├── npm@7.3.0
└── yarn@1.22.10
```
this, mind you, is after i already rimraf'd `~/.npm/_npx`

**Comment by MylesBorins** [comment, marked as answer]
If there are still actionable questions here an issue should be opened at https://github.com/npm/cli/issues
